mailpoet/lib/Settings/SettingsController.php

```diff
@@ -12,7 +12,7 @@ class SettingsController {
   const DEFAULT_SENDING_METHOD = 'PHPMail';
   const DEFAULT_SENDING_FREQUENCY_EMAILS = 25;
   const DEFAULT_SENDING_FREQUENCY_INTERVAL = 5; // in minutes
-  const DEFAULT_DEACTIVATE_SUBSCRIBER_AFTER_INACTIVE_DAYS = 180;
+  const DEFAULT_DEACTIVATE_SUBSCRIBER_AFTER_INACTIVE_DAYS = 365;
```

Comment:
I considered changing this to be a string instead of an integer because it's a top level setting, i.e. it isn't inside an array. When array settings are saved to the database they're serialized, preserving type information on retrieval, but top level settings always get returned as a string and we need to be careful about type casting them in order to achieve consistent behavior.

At the end of the day I decided to leave it as an integer because that's what it is, and there are plenty of other places in our codebase where we need to typecast settings retrieved from the database. If we want to make a change it would probably be best to come up with a generic solution and apply it to all cases, but I'm curious to get the opinion of the reviewer(s).